core/CronArchive.php

```diff
@@ -716,7 +722,13 @@ private function archiveSingleSite($idSite, $requestsBefore)
         /**
          * Trigger archiving for non-day periods
          */
-        $success = $this->processArchiveForPeriods($idSite, $lastTimestampWebsiteProcessedPeriods);
+        try {
+            $success = $this->processArchiveForPeriods($idSite, $lastTimestampWebsiteProcessedPeriods);
+        } catch (UnexpectedWebsiteFoundException $e) {
+            // this website was deleted in the meantime
```

Comment:
do we need a `$this->skipped++;` here too?

Reply:
I doin't think so (after looking through), this is only used for the logger to tell how many sites were skipped.

Reply:
yeah just thought because we kind of skip the website here... at least the "log info" says "skipped" so assumed it might be needed but indeed may not be needed